test/envoye2e/http_metadata_exchange/testoutput/client.yaml

```diff
@@ -102,7 +106,7 @@ static_resources:
             address:
               socket_address:
                 address: 127.0.0.1
-                port_value: 20083
+                port_value: 20103
```

Comment:
What is this change for?

Reply:
??? generated file. Got changed by running that test.